stock_intercompany/models/stock_picking.py

```diff
@@ -0,0 +1,33 @@
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def create_counterpart_picking(self):
```

Comment:
```suggestion
    def _create_counterpart_picking(self):
```

this is an internal method, which should not be exposed to RPC